SSOUrlGenerator.php

```diff
@@ -51,4 +51,14 @@ public function generateUrl($redirectUri, $options = [])
 
         return sprintf('https://%s/authorize?%s', $this->auth0Domain, http_build_query($query));
     }
+
```

Comment:
I don't think it makes sense for the URL generator to have these getters, so remove them...